src/utils/getEntryFiles.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ *
+ * @flow
+ */
+
+const path = require('path');
+
+const handleMultipleEntries = (multiEntry: *) => {
```

Comment:
```js
(multiEntry: Array<string> | {[key: string]: string} | string)
```

Reply:
I struggled with flow when I tried to type this entry, so I ended up using asterix